firewood/src/merkle/trie_hash.rs

```diff
@@ -4,18 +4,15 @@
 use shale::{CachedStore, ShaleError, Storable};
 use std::{
     fmt::{self, Debug},
-    io::{Cursor, Write},
+    io::Write,
 };
 
 pub const TRIE_HASH_LEN: usize = 32;
+const U64_TRIE_HASH_LEN: u64 = TRIE_HASH_LEN as u64;
 
 #[derive(PartialEq, Eq, Clone)]
 pub struct TrieHash(pub [u8; TRIE_HASH_LEN]);
 
-impl TrieHash {
-    const MSIZE: u64 = 32;
```

Comment:
Wasn't pub anyway... previous code was fragile and `MSIZE` and `TRIE_HASH_LEN` could have diverged.